GDJS/Runtime/CustomRuntimeObjectInstanceContainer.ts

```diff
@@ -293,21 +300,6 @@ namespace gdjs {
       this._customObject.onChildrenLocationChanged();
     }
 
-    /**
-     * Triggered when the object dimensions are changed.
-     *
-     * It adapts the layers camera positions.
-     */
-    onObjectUnscaledCenterChanged(oldOriginX: float, oldOriginY: float): void {
```

Comment:
Nice! Why don't we need this anymore?

Reply:
The internal camera of custom objects was trying to stay at the center of the children (a bit like camera stay on the upper left corner of the scene). This way the transformation has no effect and children were not shifted. This system has been removed to avoid any side effects as it doesn't really make sense.